### DISCUSSION TITLE
how can i run Rtabmap on ROS2

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1075] [General]
hi
i installed Rtabmap on Ros2 humble using this

sudo apt install ros-humble-rtabmap-ros

now when i want to run it i'm using this:
ros2 launch rtabmap_launch rtabmap.launch.py

but it gives me this error: "launch file not found"

how can i run rtabmap on ROS2 ?

**Comment by matlabbe** [comment]
Humm try re-source ros setup.bash

**Comment by hamdi-sharaf** [comment]
which version of ros2 you are using? and with witch hardware you are using?

**Reply by damavand1** [reply]
i'm using ros2 humble, and run ubuntu 22.04 sometimes on RPI 4 4GB and sometimes on my laptop

Problem solved using "re-source ros setup.bash"
